tests/framework-tests/preferences/js/PanelsTests.js

```diff
@@ -1043,6 +1049,8 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         }
     });
 
+    fluid.tests.prefs.panel.templatePrefix = "../../../../src/framework/preferences/html/";
```

Comment:
This variable declaration can be moved a few lines above before line 1009 so the calculation of the location for the text font template can make use of it too.